gnomad/utils/transcript_annotation.py

```diff
@@ -0,0 +1,59 @@
+"""Utils module containing generic functions that are useful for adding transcript expression-aware annotations."""
+from typing import Callable, Optional
+
+import hail as hl
+
+
+def summarize_rsem_mt(
+    rsem_mt: hl.MatrixTable,
+    rsem_expr: hl.expr.NumericExpression,
+    tissue_expr: hl.expr.StringExpression,
```

Comment:
Since rsem_expr will be 'mt.x' and tissue_expr will 'mt.tissue' in the resource mt I'm making, I'm not sure if it's still necessary to define them as arguments.